integration_tests/README.md

```diff
@@ -23,6 +23,7 @@ To update the version of Daphne in use:
 1. Update `Cargo.toml` in this directory to reference the new commit of Daphne.
 1. Update `build.rs` in this directory to reference the new commit of Daphne.
 1. Update this README to note the new commit of Daphne.
+1. Build a new prebuilt image as described below, upload it, and update secrets for both GitHub Actions and Dependabot to use the new image.
```

Comment:
This documentation should be phrased "If using a prebuilt image..."

The references to GH Actions & Dependabot should be dropped, since those are specific to our particular GH repo.

Reply:
(I'm working on a more detailed set of step-by-step instructions for our infra, going into our private documentation.)